mcs/class/referencesource/mscorlib/system/type.cs

```diff
@@ -1249,7 +1249,6 @@ public virtual Type[] GetGenericParameterConstraints()
             get {return IsCOMObjectImpl();}
         }
 
-#if FEATURE_COMINTEROP || MONO_COM
```

Comment:
What do you suggest I do then? The other reference source code is calling into this method, and it does not compile when MONO_COM is not defined.